.github/workflows/ab_tests.yml

```diff
@@ -69,7 +68,6 @@ jobs:
           miniforge-variant: Mambaforge
           use-mamba: true
           condarc-file: ci/condarc
-          python-version: ${{ matrix.python-version }}
```

Comment:
I could be missing something, but it looks like `python` is unpinned in `ci/environment.yml`

https://github.com/coiled/benchmarks/blob/278d82b3e282f59e97ed789c82f5ef018ff6c69b/ci/environment.yml#L8

My guess is there's another environment file that comes into play here, but just wanted to double check

Reply:
Ah. I just realised. I'm reworking and retesting.